tests/testthat/test-available-packages.R

```diff
@@ -70,7 +70,6 @@ test_that("available_packages() succeeds with unnamed repositories", {
 test_that("renv_available_packages_latest() respects pkgType option", {
 
   skip_on_cran()
-  skip_if(.Platform$pkgType == "source")
```

Comment:
Can you check if this test succeeds if you run:

```
platform <- .Platform
platform$pkgType <- "source"
renv:::renv_binding_replace(".Platform", platform, envir = baseenv())
```

before running this test? IIRC I ran into some issues here when using R from Homebrew / compiled from sources for some reason.

Reply:
You're right, it fails. I'm not sure why I removed it, but I'll restore it.